pkg/network/usm/compile.go

```diff
@@ -18,7 +18,7 @@ import (
 //go:generate $GOPATH/bin/integrity pkg/ebpf/bytecode/build/runtime/usm.c pkg/ebpf/bytecode/runtime/usm.go runtime
 
 func getRuntimeCompiledUSM(config *config.Config) (runtime.CompiledOutput, error) {
-	return runtime.Usm.Compile(&config.Config, getCFlags(config), statsd.Client)
+	return runtime.Usm.Compile(&config.Config, getCFlags(config), getLlcFlags(), statsd.Client)
```

Comment:
why not simply passing nil?
I saw you did it from oom_kill and tcp_queue_length

Reply:
Follow up PRs will introduce a flag to be passed to `llc`